### DISCUSSION TITLE
经典书籍推荐 | HackWay技术学习路线

### DISCUSSION THREAD

**Discussion by giscus** [lidongyx/hackwaydoc#46] [Announcements]
# 经典书籍推荐 | HackWay技术学习路线

计算机相关的书籍很多，网上免费的电子资源也是十分丰富，但是学习务必求精。广泛翻阅了大量的计算机书籍推荐的文章，包括国内以及国外的推荐，考虑到通用性，对于计算机专业的学生来说，只推荐最为经典必买的书，这些书大多都是国内外诸多顶级名校的本科生教材，而且一版再版，不少书籍的作者还是计算机最高奖项图灵奖的获得者。

https://hackway.org/docs/cs/intro

**Comment by whoamiyan** [comment]
很赞！

**Comment by XJYdemons** [comment]
谭浩强哈哈哈，垃圾谭浩强，害了多少人
